issue#159/git_scraper.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""git_scraper.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1vVkTt6t8CcCS4tXkAsfDVjD9BkoYyqPC
+"""
+
+
+
+
+
+import requests
```

Comment:
Again, why are you adding it as a separate file? Add it as a method of the `Users` class (of the GitHub module).

Check how the GitHub module is organized and how classes and methods work.